buildenv/jenkins/jobs/infrastructure/Update-Reference-Repos.groovy

```diff
@@ -100,7 +98,7 @@ timeout(time: 6, unit: 'HOURS') {
                         jobs["${sNodeName}"] = {
                             node("${sNodeName}"){
                                 stage("${sNodeName} - Update Reference Repo") {
-                                    refresh(sNodeName, "${HOME}/openjdk_cache", EXTENSIONS_REPOS, true)
+                                    refresh(sNodeName, "${HOME}/openjdk_cache", [name: "openj9", url: VARIABLES.openj9.get('default').get('repoUrl')], true)
```

Comment:
`refresh()` expects an array, not a map: see https://github.com/eclipse/openj9/blob/e501ff270127f286a188ae64a6971855dd329ac1/buildenv/jenkins/jobs/infrastructure/Update-Reference-Repos.groovy#L184